templates/springnative/cloudevents/manifest.yaml

```diff
@@ -0,0 +1,5 @@
+builders:
```

Comment:
You don't need this file. These values are inherited from the language level `manifest.yaml`